.vscode/launch.json

```diff
@@ -43,7 +43,6 @@
       "port": 9229,
       "stopOnEntry": false,
       "sourceMaps": true,
-      "preLaunchTask": "Build",
```

Comment:
Deliberate?

Reply:
Yes...that should have been removed a while ago.

Reply:
It didn't seem useful to create a separate PR for it.